oauth/oidc.py

```diff
@@ -202,6 +202,7 @@ def exchange_code_for_login(self, app_config, http_client, code, redirect_suffix
         else:
             email_address = user_info.get("email") if user_info.get("email_verified") else None
 
+        logger.debug("Printing out decoded token: `%s`", user_info)
```

Comment:
What's this for?

Reply:
Was needed for debugging a different issue. Guess I just forgot to remove it by mistake. Will remove it now.